charts/akeyless-secure-remote-access/templates/_helpers.tpl

```diff
@@ -177,16 +177,28 @@ Get the Ingress TLS secret.
 {{- default "akeyless.io" .Values.ztpConfig.akeylessTenantUrl -}}
 {{- end -}}
 
-{{- define "akeyless-zero-trust-portal.storageSecretName" -}}
-        {{ $.Release.Name }}-storage-secret
+{{- define "akeyless-secure-remote-access.redisStorageImage" -}}
+  {{- if .Values.redisStorage.image -}}
+    image: "{{ .Values.redisStorage.image.repository }}:{{ .Values.redisStorage.image.tag }}"
+    imagePullPolicy: {{ .Values.redisStorage.image.pullPolicy }}
+  {{- else }}
+    image: "docker.io/bitnami/redis:6.2"
```

Comment:
Maybe this image version can be in the values?

Reply:
this is by the product instructions